fronts-client/src/components/FrontsEdit/FrontContent.tsx

```diff
@@ -178,127 +285,37 @@ class FrontContent extends React.Component<FrontProps, FrontState> {
 		);
 	}
 
+	/** This function manages the moving of cards between groups. */
 	public handleMove = (move: Move<TCard>) => {
-		const numberOfArticlesAlreadyInGroup = move.to.cards?.length ?? 0;
-		const hasMaxItemsAlready =
-			move.to.groupMaxItems === numberOfArticlesAlreadyInGroup;
-
-		// if we are moving an article into the standard group,
-		// or any group that is either empty or doesn't have the max items already,
-		// or if we're moving the article within the same group,
-		// then we just move the article to the location
+		const numberOfCardsInGroup = move.to.cards?.length ?? 0;
+		const targetGroupIsFull = move.to.groupMaxItems === numberOfCardsInGroup;
+
+		/**
+		 *  If:
+		 * - the target group isn't full
+		 * - or the card is already in the target group
+		 * - or it's a standard group
```

Comment:
Does this assume the `standard` is the bottom group? If so we might want to make this logic more abstract.